int8 voice # 1-100
---
string result
string fail_reason
